Don't install luet again

jimmykarily opened the pull request:
it's installed some lines above with the user specified version

jimmykarily committed:
Don't install luet again

it's installed some lines above with the user specified version

Signed-off-by: Dimitris Karakasilis <dimitris@karakasilis.me>

jimmykarily commented on main.go:
```diff
@@ -109,7 +109,6 @@ func main() {
 
 	utils.RunSH("dependencies", "mv luet /usr/bin/luet && mkdir -p /etc/luet/repos.conf.d/")
 	utils.RunSH("dependencies", "curl -L https://raw.githubusercontent.com/mocaccinoOS/repository-index/master/packages/luet.yml --output /etc/luet/repos.conf.d/luet.yml")
```
I'm not sure if we should keep this line either. This point to really old version of packages. Why would we need this at all? Do we install anything else other than luet?

mudler replied:
there are other plugins available, probably not used

jimmykarily committed:
Auto-detect buildx

so that the consumer can install whatever version of it they want. E.g.
by using: docker/setup-buildx-action Github action

Signed-off-by: Dimitris Karakasilis <dimitris@karakasilis.me>

mudler commented on main.go:
```diff
@@ -98,18 +97,8 @@ func main() {
-
 	utils.RunSH("dependencies", "mv luet /usr/bin/luet && mkdir -p /etc/luet/repos.conf.d/")
 	utils.RunSH("dependencies", "curl -L https://raw.githubusercontent.com/mocaccinoOS/repository-index/master/packages/luet.yml --output /etc/luet/repos.conf.d/luet.yml")
-	utils.RunSH("dependencies", "luet install -y system/luet")
```
shall we put this behind a flag too?

jimmykarily replied:
but this was always redundant. There is a `luetVersion` flag which already works and installs that luet version. Then we come here installing this on top.

jimmykarily committed:
Revert "Auto-detect buildx"

This reverts commit 85059c4dcaa643b889209f419dd7c3801bddc1c0.

jimmykarily committed:
Keep installation of buildx for backwards compatibility

Signed-off-by: Dimitris Karakasilis <dimitris@karakasilis.me>
